packages/libomxil_bellagio.rb

```diff
@@ -29,6 +29,7 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    # ????
```

Comment:
Huh?

Reply:
The lines below it speak for themselves..

Reply:
If we don't know why they're there, we should find out and document them-- you added those lines in a753b392ea31ae60efb8c51148567e8d17c6f224 @uberhacker, do you remember why?